inspire/utils/cv_latex.py

```diff
@@ -0,0 +1,462 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2015 CERN.
+#
+# INSPIRE is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#
+
+import re
+from invenio.config import CFG_SITE_URL
```

Comment:
I am going to change it in the other PR too.

2015-09-09 11:27 GMT+02:00 Giannis Tsanaktsidis <
ioannis.tsanaktsidis@cern.ch>:

> Fixed!
>
> 2015-09-09 11:22 GMT+02:00 Javier Martin Montull <notifications@github.com
> >:
>
>> In inspire/utils/cv_latex.py
>> <https://github.com/inspirehep/inspire-next/pull/334#discussion_r39022436>
>> :
>>
>> > +# modify it under the terms of the GNU General Public License as
>> > +# published by the Free Software Foundation; either version 2 of the
>> > +# License, or (at your option) any later version.
>> > +#
>> > +# INSPIRE is distributed in the hope that it will be useful, but
>> > +# WITHOUT ANY WARRANTY; without even the implied warranty of
>> > +# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
>> > +# General Public License for more details.
>> > +#
>> > +# You should have received a copy of the GNU General Public License
>> > +# along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
>> > +# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
>> > +#
>> > +
>> > +import re
>> > +from invenio.config import CFG_SITE_URL
>>
>> Correct way of using config is:
>>
>> from invenio.base.globals import cfg
>>
>> And then:
>>
>> cfg['CFG_SITE_URL']
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/inspirehep/inspire-next/pull/334/files#r39022436>.
>>
>
>